nginx_config_reloader/__init__.py

```diff
@@ -248,44 +246,17 @@ def wait_loop(logger=None, allow_includes=False):
             logger.warning("Configuration dir lost, waiting for it to reappear")
 
 
-def parse_nginx_config_reloader_arguments():
-    parser = argparse.ArgumentParser()
-    parser.add_argument('--daemon', '-d', action='store_true', help='Fork to background and run as daemon')
-    parser.add_argument('--monitor', '-m', action='store_true', help='Monitor files on foreground with output')
```

Comment:
we can remove the daemon mode though, so it would be either monitor or run only once.